src/Storage/Storage.Management.Test/ScenarioTests/StorageAccountTests.ps1

```diff
@@ -2115,4 +2115,72 @@ function Test-AzureStorageAccountWorm
         # Cleanup
         Clean-ResourceGroup $rgname
     }
-}
\ No newline at end of file
+}
+
+<#
+.SYNOPSIS
+Test Test-NewAzStorageContext
+.DESCRIPTION
+SmokeTest
+#>
+function Test-NewAzStorageContext
+{
+    # Setup
+    $rgname = Get-StorageManagementTestResourceName;
+
+    try
+    {
+        # Test
+        $stoname = 'sto' + $rgname;
+        $stotype = 'Standard_LRS';
+        $kind = 'StorageV2'
+        $containerName = "container"+ $rgname
+        $blobEndpoint = 'https://' + $stoname + '.blob.core.windows.net/'
+        $tableEndpoint = 'https://' + $stoname + '.table.core.windows.net/'
+        $queueEndpoint = 'https://' + $stoname + '.queue.core.windows.net/'
+        $fileEndpoint = 'https://' + $stoname + '.file.core.windows.net/'
+
+        $loc = Get-ProviderLocation ResourceManagement;
+        New-AzResourceGroup -Name $rgname -Location $loc;
+		
+        New-AzStorageAccount -ResourceGroupName $rgname -Name $stoname -Location $loc -Type $stotype -Kind $kind;
+
+        $sto = Get-AzStorageAccount -ResourceGroupName $rgname  -Name $stoname;
```

Comment:
The above $blobEndpoint ...., might can be read from Primary endpoint of the Get-AzStorageAccount result.
The change should don't need test record again. Just case change.

Reply:
Done

Reply:
Done